intents/test.py

```diff
@@ -0,0 +1,135 @@
+"""This module has some examples of function calls for queries from IPL & spider dataset."""
```

Comment:
For all test cases, have a consistent format of print statements and logic computations

Reply:
Also, ideally there should be a different test file for each of the intents.

Reply:
I'll change that